packages/jest-config/src/getMaxWorkers.ts

```diff
@@ -18,18 +18,18 @@ export default function getMaxWorkers(
     const maxWorkers = (argv.maxWorkers as unknown) as number | string;
     const parsed = parseInt(maxWorkers as string, 10);
 
+    const cpus = os.cpus().length;
```

Comment:
l35 also has a `os.cpus()`, can move this out of the if and reuse it there. Also I think it's important to do the `os.cpus() ?` check like l35 does because it can be `undefined` e.g. on Android.